Mathlib/RepresentationTheory/Action/Concrete.lean

```diff
@@ -79,6 +87,74 @@ theorem ofMulAction_apply {G : Type u} {H : FintypeCat.{u}} [Monoid G] [MulActio
     (g : G) (x : H) : (FintypeCat.ofMulAction G H).ρ g x = (g • x : H) :=
   rfl
 
+section
+
+/-- Shorthand notation for the quotient of `G` by `H` as a finite `G`-set. -/
+notation:10 G:10 " ⧸ₐ " H:10 => Action.FintypeCat.ofMulAction G (FintypeCat.of <| G ⧸ H)
+
+variable {G : Type*} [Group G] (H N : Subgroup G) [Fintype (G ⧸ N)]
+
+/-- If `H` and `N` are subgroups of a group `G` with `N` normal, there is a canonical
+group homomorphism `H ⧸ N ⊓ H` to the `G`-endomorphisms of `G ⧸ N`. -/
+def quotientToEndHom [N.Normal] : H ⧸ Subgroup.subgroupOf N H →* End (G ⧸ₐ N) :=
+  let φ' : H →* End (G ⧸ₐ N) := {
+    toFun := fun ⟨v, _⟩ ↦ {
+      hom := Quotient.lift (fun σ ↦ ⟦σ * v⁻¹⟧) <| fun a b h ↦ Quotient.sound <| by
+        apply (QuotientGroup.leftRel_apply).mpr
+        simp only [mul_inv_rev, inv_inv]
+        convert_to v * (a⁻¹ * b) * v⁻¹ ∈ N
+        · group
+        · exact Subgroup.Normal.conj_mem ‹_› _ (QuotientGroup.leftRel_apply.mp h) _
+      comm := fun (g : G) ↦ by
+        ext (x : G ⧸ N)
+        induction' x using Quotient.inductionOn with x
+        simp only [FintypeCat.comp_apply, Action.FintypeCat.ofMulAction_apply, Quotient.lift_mk]
+        show Quotient.lift (fun σ ↦ ⟦σ * v⁻¹⟧) _ (⟦g • x⟧) = _
+        simp only [smul_eq_mul, Quotient.lift_mk, mul_assoc]
+        rfl
+    }
+    map_one' := by
+      apply Action.hom_ext
+      ext (x : G ⧸ N)
+      induction' x using Quotient.inductionOn with x
+      simp
+    map_mul' := fun σ τ ↦ by
+      apply Action.hom_ext
+      ext (x : G ⧸ N)
+      induction' x using Quotient.inductionOn with x
+      show ⟦x * (σ * τ)⁻¹⟧ = ⟦x * τ⁻¹ * σ⁻¹⟧
+      rw [mul_inv_rev, mul_assoc, Subgroup.coe_mul]
+  }
```

Comment:
I think we should have this map as a standalone definition for the case `G ->* End`. Then you can compose that map with the inclusion `H -> G` inside this definition.

Reply:
Done